### DISCUSSION TITLE
求更新

### DISCUSSION THREAD

**Discussion by ghost** [win12-online/win12#325] [General]
请问何时再次更新？

**Comment by User782Tec** [comment]
预计一个星期
我们正在加紧开发qwq
这次更新会添加:
桌面小组件
桌面的编辑模式
windows更新完善
更新机制重写
浏览器和文件资源管理器的前进和后退功能
修复若干bug
......

**Comment by ghost** [comment]
OK，请尽快更新。
@User782Tec

**Comment by ghost** [comment]
不过编辑模式具体指啥？

**Comment by User782Tec** [comment]
就是可以移动桌面小组件的入口

**Comment by User782Tec** [comment]
> OK，请尽快更新。 @User782Tec

好哒
感谢支持

**Comment by ghost** [comment]
@User782Tec 那必须的(我是你们的粉丝~)

**Comment by User782Tec** [comment]
哈哈 非常感谢你的支持

**Comment by ghost** [comment]
不用谢
